mkdd_editor.py

```diff
@@ -1479,7 +1481,7 @@ def get_recent_files_list(self):
         return recent_files
 
     #@catch_exception
-    def button_load_level(self, checked=False, filepath=None):
+    def button_load_level(self, checked=False, filepath=None, add_to_ini = True):
```

Comment:
`add_to_ini` may not be a good name. It exposes the implementation details. Imagine if we needed to change the underlying format from INI to JSON/TOML/YAML at some point; we wouldn't want to have to rename a parameter in a function.

My proposal would be `update_recent_files_list`, `add_to_recent_files_list` or `update_config`, or something on that note.

Also, default parameters are not expected to have spaces between the parameter name and the default value:

```diff
- parameter_name = Value
+ parameter_name=Value
```
This is mentioned in [PEP 8 > Other Recommendations](https://peps.python.org/pep-0008/#other-recommendations).